internal/command/organization_settings.go

```diff
@@ -0,0 +1,72 @@
+package command
+
+import (
+	"context"
+
+	"github.com/zitadel/zitadel/internal/domain"
+	"github.com/zitadel/zitadel/internal/zerrors"
+)
+
+type SetOrganizationSettings struct {
+	OrganizationID string
+
+	UserUniqueness *bool
+}
+
+func (e *SetOrganizationSettings) IsValid() error {
+	if e.OrganizationID == "" {
+		return zerrors.ThrowInvalidArgument(nil, "COMMAND-TODO", "Errors.Org.Settings.Invalid")
+	}
+	return nil
```

Comment:
Is a request without any change request even valid? I totally agree that a request without affect is valid, but if there's not even the intent on changing something (all fields unset), i would consider it an invalid request.

Reply:
Hm good question, I looked at it as valid as the attribute value is optional, and then if no changes are done no event is emitted. Would that then be the case for all v2 APIs?